### ISSUE TITLE
A11y_chat window_HighContrast: In aquatic and desert modes, 'Retry' control is not visually appearing as interactive control.

### ISSUE THREAD

**Issue by compulim** [microsoft/BotFramework-WebChat#4558]
### Is it an issue related to Adaptive Cards?

No.

### What browsers and screen readers do this issue affect?

Windows: Edge with Windows Narrator, Windows: Chrome with NVDA, Windows: Chrome/Firefox with JAWS

### Are there any code-based customization done to Web Chat?

No, I am using Web Chat without any customizations except "styleOptions".

### What version of Web Chat are you using?

Latest production

### Which area does this issue affect?

Contrast ratio, Forced colors (high contrast mode)

### How to reproduce the issue?

1. On Windows, turn on high contrast mode with a light theme, such as Aquatic or Desert
2. Navigate to https://compulim.github.io/webchat-loader/ and start a chat
3. Turn on airplane mode
4. Type "hello" to the bot
5. Wait until it failed to send

### What do you expect?

The "Send failed. Retry." message should clearly show "Retry" is a link by using the link color defined in Aquatic or Desert high contrast mode.

### What actually happened?

The "Retry" button is not clearly shown as a link button.

### Do you have any screenshots or recordings to repro the issue?

![image](https://user-images.githubusercontent.com/1622400/206584703-e0b6b542-8014-43f3-8b57-3c614ad2d7d4.png)

![image](https://user-images.githubusercontent.com/1622400/206584654-90832197-6af1-438a-94cb-1f072576e5f8.png)

![image](https://user-images.githubusercontent.com/1622400/206584686-a45fc0f4-7e3f-4770-a1b6-5cca88047da2.png)

### Additional context

In High Contrast mode, the "Retry" button/link should be in hyperlink color

https://msazure.visualstudio.com/CCI/_workitems/edit/15603122
